templates/vhost/_http2.epp

```diff
@@ -0,0 +1,45 @@
+<% if $h2_copy_files != undef {-%>
+  H2CopyFiles <%= apache::bool2httpd($h2_copy_files) %>
+<% } -%>
+<% unless $h2_direct == undef {-%>
+  H2Direct <%= apache::bool2httpd($h2_direct) %>
+<% } -%>
+<% if $h2_early_hints != undef {-%>
```

Comment:
I think this style is a bit confusing. One uses `if` and the other `unless`, while both doing the same thing. I'd align on `if $param != undef` consistently throughout this file.

Reply:
or unless in whole file ... i think that would be minimal changes ..thoughts ?

Reply:
+1 for using the same syntax to "output this if $foo is not undef"

Reply:
@ekohl  done with changes so marking as resolved.